packages/dashboard/src/DashboardPlugin.ts

```diff
@@ -8,6 +8,11 @@ import type {
 } from '@deephaven/golden-layout';
 import PanelManager from './PanelManager';
 
+export type PanelFunctionComponentType<P> = NamedExoticComponent<P> & {
+  COMPONENT?: string;
+  TITLE?: string;
```

Comment:
Since this is used here and in `PanelComponentType`, may want to separate it out and add some JSDocs about what each is for